plugins/module_utils/storage/dell/utils.py

```diff
@@ -123,6 +124,12 @@ def get_powermax_management_host_parameters(metro_dr=False):
         port=dict(type='int', required=False, default=8443))
 
 
+def get_powermax_management_host_parameters_remove_sensitive_data(host_dict):
+    copy_dict_value = copy.deepcopy(host_dict)
+    copy_dict_value.pop('password')
```

Comment:
As we have marked [no_log](https://github.com/dell/ansible-powermax/pull/76/files#diff-ba25a9a1aa0ae44a71e630e90cec954e68bc6ea6568861d2f33e9ef262522fabL120) as true for password. Still, we need this.
[Reference](https://docs.ansible.com/ansible/latest/reference_appendices/logging.html#protecting-sensitive-data-with-no-log)